cmd/gaurun/gaurun.go

```diff
@@ -38,7 +38,7 @@ func main() {
 	gaurun.ConfGaurun = gaurun.BuildDefaultConf()
 
 	// load configuration
-	conf, err := gaurun.LoadConf(gaurun.ConfGaurun, *confPath)
+	conf, err := gaurun.LoadConf(*confPath)
```

Comment:
This change breaks the compatibility of default value.

https://github.com/mercari/gaurun/blob/master/gaurun/conf.go#L67-L98

Reply:
I fixed it.